config/env.properties

```diff
@@ -1,3 +1,4 @@
+## Properties associated with the environment should go in this file.
```

Comment:
can you look at this file and env.properties at the HEAD of master in camel-boot?  I moved these from file to file and added a few more comments.